cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControl.java

```diff
@@ -738,8 +738,8 @@ public void executeDemotion(Set<ExecutionProposal> proposals,
    *
    * @param forceExecutionStop Whether force execution to stop.
    */
-  public void userTriggeredStopExecution(boolean forceExecutionStop) {
-    _executor.userTriggeredStopExecution(forceExecutionStop);
+  public void userTriggeredStopExecution(boolean forceExecutionStop, boolean stopExternalAgent) {
```

Comment:
Missing `@param` in JavaDoc for `stopExternalAgent`. 

**Note**: We should enforce this via CheckStyle, but currently there are lots of violations on this on the legacy code. So, this is another style issue in the pipeline -- would you mind creating an issue about it to keep track of it?